src/renderer/components/Login.js

```diff
@@ -206,6 +211,17 @@ export default class Login extends React.Component {
               <option value='starttls'>STARTTLS</option>
               <option value='plain'>{tx('off')}</option>
             </DeltaSelect>
+            <DeltaSelect
+              id='imap_certificate_checks'
+              label={tx('login_imap_certificate_checks')}
```

Comment:
I think the entry for login_imap_certificate_checks is missing in the sources, or?

Reply:
Yup seems like the translations (at least experimental/untranslated) are missing.